readme.md

```diff
@@ -84,6 +106,12 @@ Type: `any`
 
 Value to resolve or reject in the returned promise.
 
+#### signal
```

Comment:
I would call it `abortSignal`. Just `signal` is too generic.

Reply:
Yeah for a parameter I agree, for an options object I would follow the native APIs:

> Any web platform API using promises to represent operations that can be aborted must adhere to the following:
>
> - Accept AbortSignal objects through a `signal` dictionary member.

https://dom.spec.whatwg.org/#abortcontroller-api-integration

I.e. this is what fetch and WebAuth do.

Reply:
👍 Alright then.